README.md

```diff
@@ -165,12 +167,13 @@ If you work with Visual Studio Code, please install these three extensions:
 - https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig
 - https://marketplace.visualstudio.com/items?itemName=astro-build.astro-vscode
 
-Auto-format on save is activated via `.vscode/settings.json` for Visual Studio Code.
+Autoformat on save is activated via `.vscode/settings.json` for Visual Studio Code.
```

Comment:
```suggestion
[Automatically "Format on Save" setting](vscode://settings/editor.formatOnSave) is activated via `.vscode/settings.json` for Visual Studio Code.
```

- This links directly to the settings in normal markdown and HTML (when converted to anchor). 

It appears broken in GitHub preview. Not sure if Astro will have problems transform it either.

Reply:
Thanks for flagging this. I don't think the GitHub rendering can handle a vscode scheme, so we probably shouldn't do it here.

Reply:
```suggestion
'Automatically Format on Save' setting is activated via `.vscode/settings.json` for Visual Studio Code.
```